supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -359,6 +359,50 @@ There is no functional difference between the first server that was installed an
 
 *See also*:
 
+[[inference]]
+==== image:images/yes.png[yes] inference (noun)
+*Description*: Inference is when a model can process, deduct, and produce outputs based on input data. The process of using a trained AI model to generate predictions or conclusions based on the input data provided to the model.
+
+*Use it*: yes
+
+[.vale-ignore]
+*Incorrect forms*: 
+
+*See also*:
+
+[[`InferenceService`]]
```

Comment:
This is messing up the formatting

![image](https://github.com/user-attachments/assets/65945550-36e2-459e-b950-3a6adf9814ff)

Should just be

```suggestion
[[inferenceservice]]
```